samples/kafka_confluent/receiver-assign/main.go

```diff
@@ -0,0 +1,60 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	kafka_confluent "github.com/cloudevents/sdk-go/protocol/kafka_confluent/v2"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/client"
+	"github.com/confluentinc/confluent-kafka-go/v2/kafka"
```

Comment:
nit: if you use import alias `confluent` code will be `confluent.New` and easier to read

Reply:
Done